decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -143,6 +162,40 @@ def result_published_questions
         []
       end
 
+      def to_json(admin: false)
```

Comment:
shouldn't be this a presenter class?

Reply:
Let's use a presenter, and then find the time to actually implement this as a strategy for all the classes (with all the fallout that could come). 

Personal note: A problem with the Decidim code that i have observed over the years, is that we all have tried different approaches on the code style, but we never managed to get it through all the modules. This is lead to inconsistencies and much more tech debt than it should :(